models/nft/avalanche_c/nft_avalanche_c_transfers.sql

```diff
@@ -0,0 +1,62 @@
+ {{ config( alias='transfers') }}
+
```

Comment:
SInce nft_transfers views are updated incrementally by chain(as shown in dbt_project.yml),
we can fill more information in the config blocks for models under different chains.
```suggestion
{{ config(
        alias ='transfers',
        partition_by='block_date',
        materialized='incremental',
        file_format = 'delta',
        post_hook='{{ expose_spells(\'["avalanche_c"]\',
                                    "sector",
                                    "nft",
                                    \'["hildobby"]\') }}',
        unique_key = ['unique_transfer_id']
)
}}
```